apps/server/src/modules/tool/uc/school-external-tool.uc.ts

```diff
@@ -5,12 +5,14 @@ import { AuthorizationService } from '@src/modules/authorization';
 import { AllowedAuthorizationEntityType } from '@src/modules/authorization/interfaces';
 import { SchoolExternalToolService } from '../service/school-external-tool.service';
 import { SchoolExternalToolQueryInput } from './dto/school-external-tool.types';
+import { CourseExternalToolService } from '../service/course-external-tool.service';
 
 @Injectable()
 export class SchoolExternalToolUc {
 	constructor(
 		private readonly authorizationService: AuthorizationService,
-		private readonly schoolExternalToolService: SchoolExternalToolService
+		private readonly schoolExternalToolService: SchoolExternalToolService,
+		private readonly courseExternalToolService: CourseExternalToolService
```

Comment:
I'd like to talk to you guys about code duplication

Reply:
Yes, lets talk about that tomorrow 👍